types/bero/tslint.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "npm-naming": [
+      true,
+      {
+        "mode": "code",
+        "errors":[
+          [
+            "NeedsExportEquals",
+            false
+          ]
+        ]
+      }
+    ]
+  }
```

Comment:
Are the two overrides here a copy & paste error?

Reply:
@orta it is not in progress, for me it is the final version, possibly to be merged

Reply:
👍🏻  sure